packages/cli-kit/src/node/error-handler.test.ts

```diff
@@ -44,3 +46,58 @@ describe('errorHandler', () => {
     expect(process.exit).toBeCalledWith(1)
   })
 })
+
+beforeEach(() => {
+  vi.mock('./output')
+})
+
+describe('outputError', () => {
+  it('error output uses same input error instance when the error type is abort', async () => {
+    // Given
+    const abortError = new Abort('error message', 'try message')
+    const outputMock = outputMocker.mockAndCaptureOutput()
+
+    // When
+    await outputError(abortError)
+
+    // Then
+    expect(unstyled(outputMock.error())).toMatchInlineSnapshot(`
+      "
+      ━━━━━━ Error ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+          error message
+
+          What to try:
+          try message
+
+      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+      "
+    `)
+    outputMock.clear()
+  })
+
+  it('error output uses same input error instance when the error type is bug', async () => {
+    // Given
+    const bugError = new Bug('error message', 'try message')
+    const outputMock = outputMocker.mockAndCaptureOutput()
+
+    // When
+    await outputError(bugError)
+
+    // Then
+    expect(unstyled(outputMock.error())).not.toBe('')
```

Comment:
I tried few approaches here and ultimately landed on this but I'm not quite happy. I initially used `toMatchInlineSnapshot` but because it includes the stacktraces it makes the test dependent on the environment. Then I tried to remove the environment-dependent lines from the string to match the rest of the content but due to the indentations and the spaces it failed to match too. I'm open to suggestions.